src/EndOfYearBanner/index.stories.tsx

```diff
@@ -4,10 +4,17 @@ import { StorybookWrapper } from '../utils/StorybookWrapper';
 import { knobsData } from '../utils/knobsData';
 import { coreArgTypes } from '../storybookCommon/argTypes';
 import { BrazeMessageProps } from '.';
+import { grid, withGrid } from '../../.storybook/grid/withGrid';
 
 export default {
-    component: 'TheGuardianIn2020Banner',
-    title: 'Banner/TheGuardianIn2020Banner',
+    component: 'EndOfYearBanner',
+    title: 'Banner/EndOfYearBanner',
```

Comment:
I'd say maybe this should be in the work-in-progress folder until this PR is merged, a new `@guardian/braze-components` version is released and is available on frontend/DCR. What do you reckon? I think that's how we've done things in the past (we should add this to the README if we agree that flow makes sense).

Reply:
Yep this is now in WorkInProgress here: 
https://github.com/guardian/braze-components/blob/end-of-year-banner/src/BannerWithLink/index.stories.tsx#L11